src/features/pages/home/live-pricing/table-component/live-pricing.module.scss

```diff
@@ -63,7 +63,7 @@
     gap: 1rem;
 
     td {
-        padding: 1rem;
+        padding-block: 1rem;
```

Comment:
Why is padding-block using here instead of `padding: 1rem 0;`? Is it padding-block better?

Reply:
hi @jia-deriv  padding bock is different  its CSS logical properties,
I had to give padding only for top and bottom so I did this